int32 id
UserJoint[] joints
bool isTracking
